jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/SerializableMessageProducerPool.java

```diff
@@ -15,111 +15,65 @@
-                                                                                                                 // requests.
-      props.put("retries",
-            odeProperties.getProperty("kafka.producer.retries", MessageProducer.DEFAULT_PRODUCER_RETRIES)); // If
-                                                                                                            // the
-                                                                                                            // request
-                                                                                                            // fails,
-                                                                                                            // the
-                                                                                                            // producer
-                                                                                                            // can
-      // automatically retry
-      props.put("batch.size",
-            odeProperties.getProperty("kafka.producer.batch.size", MessageProducer.DEFAULT_PRODUCER_BATCH_SIZE_BYTES));
-      props.put("linger.ms",
-            odeProperties.getProperty("kafka.producer.linger.ms", MessageProducer.DEFAULT_PRODUCER_LINGER_MS));
-      props.put("buffer.memory", odeProperties.getProperty("kafka.producer.buffer.memory",
-            MessageProducer.DEFAULT_PRODUCER_BUFFER_MEMORY_BYTES));
-      props.put("key.serializer",
-            odeProperties.getProperty("kafka.key.serializer", MessageProducer.SERIALIZATION_STRING_SERIALIZER));
-      props.put("value.serializer",
-            odeProperties.getProperty("kafka.value.serializer", MessageProducer.SERIALIZATION_BYTE_ARRAY_SERIALIZER));
-
-      return this;
-   }
-
-   @Override
-   protected MessageProducer<K, V> create() {
-      return new MessageProducer<>(brokers, type, partitionerClass, props,
-            odeKafkaProperties.getDisabledTopics());
-   }
-
-   @Override
-   public boolean validate(MessageProducer<K, V> o) {
-      return o.getProducer() != null;
-   }
-
-   @Override
-   public void expire(MessageProducer<K, V> o) {
-      o.close();
-   }
-
-   public String getBrokers() {
-      return brokers;
-   }
-
-   public void setBrokers(String brokers) {
-      this.brokers = brokers;
-   }
-
-   public String getType() {
-      return type;
-   }
-
-   public void setType(String type) {
-      this.type = type;
-   }
-
-   public Properties getProps() {
-      return props;
-   }
-
-   public void setProps(Properties props) {
-      this.props = props;
-   }
-
-   public String getPartitionerClass() {
-      return partitionerClass;
-   }
+import java.util.Properties;
 
-   public void setPartitionerClass(String partitionerClass) {
-      this.partitionerClass = partitionerClass;
-   }
+@Getter
+@Setter
+public class SerializableMessageProducerPool<K, V> extends SerializableObjectPool<MessageProducer<K, V>> {
```

Comment:
These annotations auto-generate getters and setters for the properties in SerializableMessageProducerPool. After looking at the class and the need for these getters/setters, I've realized they really add no value to this class. I'm removing the annotations and updating the tests, since the tests were the only ones referencing the getters (setters weren't even in use).

Reply:
updated: https://github.com/CDOT-CV/jpo-ode/pull/113/commits/e3f1e3f7bcd915ff7b80a09cd7a425d0fe00ac42